### ISSUE TITLE
asksign / integrate fails in Maxima 5.30.0

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3293]
Imported from SourceForge on 2024-07-07 19:54:48
Created by **[wihaa](https://sourceforge.net/u/wihaa/)** on 2013-08-29 16:58:51
Original: https://sourceforge.net/p/maxima/bugs/2627

---

Supposed bug in Maxima 5.30.0

An answer for the request for a case differentiation
in the following expression is not possible
(neither in wxMaxima nor in the command line program):

~~~~
(%i1) integrate(sin(omega*t)*exp(-s*t),t,0,inf);
Is s positive, negative or zero?
~~~~

An answer is not accepted (neither n, nor z, nor p).
The old Maxima version 5.27.0 works fine.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 19:54:50
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2013-09-03 00:51:46
Original: https://sourceforge.net/p/maxima/bugs/2627/#fa12

---

- **labels**:  --> Lisp Core - Integration, Lisp Core - Assume
- **summary**: ase differentiation for integrate fails in Maxima 5.30.0 --> asksign / integrate fails in Maxima 5.30.0

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 19:54:54
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2013-09-03 00:51:51
Original: https://sourceforge.net/p/maxima/bugs/2627/#51ea

---

Revised summary to be more informative (incidentally fixing typographical error). Added tags.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 19:54:57
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2013-09-22 19:32:00
Original: https://sourceforge.net/p/maxima/bugs/2627/#a3c9

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -4,9 +4,10 @@
 in the following expression is not possible
 (neither in wxMaxima nor in the command line program):
 
-
+~~~~
 (%i1) integrate(sin(omega*t)*exp(-s*t),t,0,inf);
 Is s positive, negative or zero?
+~~~~
 
 An answer is not accepted (neither n, nor z, nor p).
 The old Maxima version 5.27.0 works fine.

~~~~

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-07 19:55:00
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2013-09-22 19:32:01
Original: https://sourceforge.net/p/maxima/bugs/2627/#aefd

---

Fixed by commit d4db75dd78c0c66.
